authorizerd.go

```diff
@@ -481,22 +481,30 @@ func (a *authority) VerifyRoleCert(ctx context.Context, peerCerts []*x509.Certif
 		return nil
 	}
 
-	var dr []string
 	drcheck := make(map[string]struct{})
 	domainRoles := make(map[string][]string)
+	addDomainRoles := func(dr []string) {
+		if len(dr) != 2 {
+			return
+		}
+		domain, roleName := dr[0], dr[1]
+		// duplicated role check
+		if _, ok := drcheck[domain+roleName]; !ok {
+			domainRoles[domain] = append(domainRoles[domain], roleName)
+			drcheck[domain+roleName] = struct{}{}
+		}
+	}
+
 	for _, cert := range peerCerts {
+		subj := cert.Subject.CommonName
+		dr := strings.SplitN(subj, ":role.", 2)
```

Comment:
Please help to define `":role."` as a constant.

https://github.com/yahoo/athenz/blob/6db634582dcfff72e2d62fd7ab31ef94dd40e87a/libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/AuthorityConsts.java#L26